NavigationBasicSample/app/src/main/java/com/example/android/navigationsample/TitleScreen.kt

```diff
@@ -33,12 +33,10 @@ class TitleScreen : Fragment() {
     ): View? {
         val view = inflater.inflate(R.layout.fragment_title_screen, container, false)
 
-        view.findViewById<Button>(R.id.play_btn).setOnClickListener {
-            Navigation.findNavController(view).navigate(R.id.action_title_screen_to_register)
-        }
-        view.findViewById<Button>(R.id.leaderboard_btn).setOnClickListener {
-            Navigation.findNavController(view).navigate(R.id.action_title_screen_to_leaderboard)
-        }
+        view.findViewById<Button>(R.id.play_btn).setOnClickListener(Navigation
+                .createNavigateOnClickListener(R.id.action_title_screen_to_register))
+        view.findViewById<Button>(R.id.leaderboard_btn).setOnClickListener(Navigation
```

Comment:
I think this modification makes it more difficult to distinguish the usage of the navigation library. Perhaps we could keep the `Navigation.createNavigate...` line on its own line?